QandA/models.py

```diff
@@ -0,0 +1,60 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class Meetings(models.Model):
+    meeting_id = models.AutoField(primary_key=True)
+    HOMER = 'H'
+    PLATO = 'P'
+    JARVIS = 'J'
+    FRIDAY = 'F'
+    VENUE_CHOICES = (
+        (HOMER,'homer'),
+        (PLATO,'plato'),
+        (JARVIS,'jarvis'),
+        (FRIDAY,'friday'),
+    )
+    venue = models.CharField(max_length=50,choices=VENUE_CHOICES,default=JARVIS)
+    moderator_id = models.ForeignKey(User, on_delete=models.CASCADE)
+    start_time = models.DateTimeField(help_text="Please use the following format: <em>YYYY-MM-DD</em>.")
+    end_time = models.DateTimeField(help_text="Please use the following format: <em>YYYY-MM-DD</em>.")
```

Comment:
- `help_text` should store what the field mean for DB not what would be shown to UI.
- We shouldn't let UI effect how we store things in DB. Application and view logic should be independent.